webui/tests/cypress/integration/02-loginout.spec.ts

```diff
@@ -4,35 +4,26 @@ describe('Sign in/out', () => {
 
   const LOGIN_ROUTE = '/det/login';
   const LOGOUT_ROUTE = '/det/logout';
+  const USERNAME_INPUT = 'input#login_username';
 
-  before(() => {
-    cy.login();
-  });
-
-  after(() => {
-    cy.login();
-  });
-
-  it('should be logged in React side', () => {
-    cy.visit('/det');
-    cy.checkLoggedIn();
+  it('should be logged out', () => {
+    cy.checkLoggedOut();
   });
 
   it('should be able to log out from React', () => {
     // Open the dropdown menu and click the button to log out.
     cy.login();
-    cy.visit('/det/dashboard');
+    cy.visit('/det');
     cy.get('#avatar').click();
     cy.get(`[role="menu"] a[href="${LOGOUT_ROUTE}"]`).should('have.lengthOf', 1);
     cy.get(`[role="menu"] a[href="${LOGOUT_ROUTE}"]`).click();
```

Comment:
**suggestion:** I found a minor "flake" where it can cause this portion to fail. What's happening is that when you click on `Sign Out` of the menu, the menu item will render with href to be `/det/logout#` with a hash tag at the end. On easy fix is to just change the matching portion to be:
```
cy.get(`[role="menu"] a[href^="${LOGOUT_ROUTE}"]`)
```
either `^=` or `*=` will work. I have this fix as part of the responsive web work, but this is such a small area, the conflict resolution should be simple.